scripts/select_appointments.sql

```diff
@@ -1 +1,17 @@
 -- retrieve appointment information
+SELECT p.family_name, a.appt_time
+FROM appointments a
+  INNER JOIN doctors d
+  ON d.id = a.doctor_id
+  INNER JOIN patients p
+  ON p.id = a.patient_id
+  WHERE a.family_name = 'Hubbard'
+;
+SELECT d.family_name, a.appt_time
+FROM appointments a
+  INNER JOIN doctors d
+  ON d.id = a.doctor_id
+  INNER JOIN patients p
+  ON p.id = a.patient_id
+  WHERE a.family_name = 'Wilcox'
+;
```

Comment:
This isn't quite right. You are referencing family_name's from the appointments table but that isn't where they exist. Check out the solution code if you need to review the correct code.